activemq-mqtt/pom.xml

```diff
@@ -199,23 +204,74 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>${surefire.version}</version>
         <configuration>
-          <forkCount>1</forkCount>
-          <reuseForks>false</reuseForks>
           <argLine>${surefire.argLine}</argLine>
           <runOrder>alphabetical</runOrder>
-           <systemPropertyValues>
-              <org.apache.activemq.default.directory.prefix>target</org.apache.activemq.default.directory.prefix>
-           </systemPropertyValues>
-           <!-- includes>
-             <include>**/*Test.*</include>
-           </includes -->
-           <excludes>
-             <exclude>**/PahoMQTNioTTest.java</exclude>
-           </excludes>
+          <reportFormat>plain</reportFormat>
+          <failIfNoTests>false</failIfNoTests>
```

Comment:
Why set this? Seems like a trap where a config change could result in no tests running and not being detected unintentionally.

Reply:
No special reason or requirement. If you prefer to leave the default, happy to do that. No strong opinion whereas it's better or not.